### PR TITLE
Add required scope in Slack docs

### PR DESCRIPTION
**Pull request by tkdrob**
## Proposed change

Add required scope in Slack docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes # https://github.com/home-assistant/core/issues/83523

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/slack.markdown [hunk 1]
```diff
@@ -24,7 +24,7 @@ The `slack` platform allows you to deliver notifications from Home Assistant to
 
 1. Create a [new app](https://api.slack.com/apps) under your Slack.com account.
 2. Click the `OAuth & Permissions` link in the sidebar, under the Features heading.
-3. In the Scopes section, add the `chat:write` scope, `Send messages as user`. If you get a `missing_scope` error when trying to send a message, check these permissions.
+3. In the Scopes section, add the `chat:write` and `dnd:read` scopes, `Send messages as user`. If you get a `missing_scope` error when trying to send a message, check these permissions.
```
Comment (bachya) [inline source/_integrations/slack.markdown]:
```suggestion
3. In the Scopes section, add the `chat:write` and `dnd:read` scopes and select `Send messages as user`. Many errors can occur if these options are not set correctly.
```

**Comment by frenck** [comment]
I don't think this PR fixed the reported issue. Documenting it is good, but the integration should handle this better. For example, it could stop retry-ing and raise an re-auth or maybe raise an repairs issue.

**Review by frenck — CHANGES_REQUESTED** [review]
Setting review status of this PR correctly.

**Comment by tkdrob** [comment]
> I don't think this PR fixed the reported issue. Documenting it is good, but the integration should handle this better. For example, it could stop retry-ing and raise an re-auth or maybe raise an repairs issue.

Let me be clear that this is not currently doing any OAuth 2 schemes like what Google/Nest has. The solution should just be for the user to add the scope in their app and redeploy it. Maybe I'm missing something?

**Comment by frenck** [comment]
> Let me be clear that this is not currently doing any OAuth 2 schemes like what Google/Nest has.

Understood 👍 
Thanks for clarifying!

../Frenck

**Review by frenck — APPROVED** [review]

**Pull request merged**